tests/Integration/SimplePieTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+// SPDX-FileCopyrightText: 2004-2023 Ryan Parman, Sam Sneddon, Ryan McCue
+// SPDX-FileCopyrightText: 2023 Art4 <art4@wlabs.de>
```

Comment:
@Art4  I can remove your credit if you want.

Although, according to [REUSE Specification – Version 3.0](https://reuse.software/spec/) _Copyright and Licensing Information_ is “the information that lists the copyright holders of a file […]” and, as I understand it, since SimplePie has no copyright assignment, you retain the copyright according to [Berne Convention](https://en.wikipedia.org/wiki/Berne_Convention).

Reply:
Yes, please remove my username and the email from the source code. I do not care much about to the mention of my name.